test/user-mutation-test.ts

```diff
@@ -0,0 +1,104 @@
+import axios from 'axios';
+import { expect } from 'chai';
+import { describe, it } from 'mocha';
+import { serverUrl } from './server-setup-test.js';
+import { prisma } from './server-setup-test.js';
+
+const mutation = {
+  query: `mutation CreateUser($createUserInput: UserInput!) {
+          createUser(input: $createUserInput) {
+            birthDate
+            email
+            id
+            name
+          }
+        }`,
+};
+
+const variables = {
+  createUserInput: {
+    name: 'usuario',
+    email: 'usuario@example.com',
+    password: '45687a',
+    birthDate: '2003-01-01',
+  },
+};
+
+describe('User Mutation Test', function () {
+  before(async () => {
+    await prisma.user.deleteMany();
+  });
+
+  afterEach(async () => {
+    await prisma.user.deleteMany();
+  });
+
+  it('should create user using axios request', async () => {
+    const responseQuery = await axios.post(
+      serverUrl,
+      { query: mutation.query, variables: variables },
+      {
+        headers: { 'Content-Type': 'application/json' },
+      },
+    );
+
+    const createdUser = responseQuery.data.data.createUser;
+    expect(createdUser).to.have.property('id');
+    expect(createdUser.name).to.equal('usuario');
+    expect(createdUser.email).to.equal('usuario@example.com');
+    expect(createdUser.birthDate).to.equal('2003-01-01');
+  });
+
+  it('it should return errors while trying to create a user with a already taken email', async () => {
```

Comment:
geralmente já consideramos o `it` do código parte da frase

```suggestion
  it('should return errors while trying to create a user with a already taken email', async () => {
```

Reply:
Mudança adicionada (o dedo escorregou nessa)